AdyenActions/Components/Redirect/BrowserComponent.swift

```diff
@@ -72,6 +73,8 @@ extension BrowserComponent: UIAdaptivePresentationControllerDelegate {
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
             if UIApplication.shared.applicationState == .active {
                 self.delegate?.didCancel()
+            } else {
+                self.delegate?.didOpenExternalApplication()
```

Comment:
that's a bit of an assumption, right?

Reply:
In a way. If user click "Cancel" and dismissed the app in less than 0.5s - same behaviour will be triggered..
Not sure if similar behaviour could be triggered by Push notification or any other system event 🤔

In general - between this and RedirectComponent that always false-fire "didFailed" on Vipps - It looks like a compromise.